internal/validation/registry_test.go

```diff
@@ -88,6 +86,26 @@ func TestValidator_Middleware(t *testing.T) {
 	}
 }
 
+func mockValidationErrors() validator.ValidationErrors {
+	type InvalidStruct struct {
+		FirstName string `json:"first_name" validate:"required"`
+	}
+
+	validate := validator.New(validator.WithRequiredStructEnabled())
+	validate.RegisterTagNameFunc(func(fld reflect.StructField) string {
+		name := strings.SplitN(fld.Tag.Get("json"), ",", 2)[0]
+		if name == "-" {
+			return ""
+		}
+		return name
+	})
+	e := validate.Struct(InvalidStruct{})
+	if e == nil {
+		return nil
+	}
+	return e.(validator.ValidationErrors)
```

Comment:
why not simply creating the struct with a message?is the error more composite?

Reply:
yes it is
but I wanted "real errors" since I need to test the function with a proper object